src/libraries/Common/tests/Tests/System/IO/StreamConformanceTests.cs

```diff
@@ -158,8 +340,8 @@ protected async Task ValidateMisuseExceptionsAsync(Stream stream)
                 Assert.Throws(UnsupportedReadWriteExceptionType, () => { stream.ReadByte(); });
                 Assert.Throws(UnsupportedReadWriteExceptionType, () => { stream.Read(new Span<byte>(new byte[1])); });
                 Assert.Throws(UnsupportedReadWriteExceptionType, () => { stream.Read(new byte[1], 0, 1); });
-                Assert.Throws(UnsupportedReadWriteExceptionType, () => { stream.ReadAsync(new byte[1], 0, 1); });
-                Assert.Throws(UnsupportedReadWriteExceptionType, () => { stream.ReadAsync(new Memory<byte>(new byte[1])); });
+                await Assert.ThrowsAsync(UnsupportedReadWriteExceptionType, () => stream.ReadAsync(new byte[1], 0, 1));
+                await Assert.ThrowsAsync(UnsupportedReadWriteExceptionType, async () => await stream.ReadAsync(new Memory<byte>(new byte[1])));
```

Comment:
Why are these two different?

Reply:
(similar below for Write)

Reply:
Ah is this because of ValueTask? Ok then.

Reply:
ThrowsAsync expects a `Func<Task>`.  The second overload returns a `ValueTask`.  So it's either using async/await or add .AsTask().  I could make the first one match with async/await, as well, just wasn't necessary.

Reply:
Right.